addons/flannel/0.20.2/install.sh

```diff
@@ -14,10 +14,11 @@ function flannel_pre_init() {
     local dst="$DIR/kustomize/flannel"
 
     if flannel_weave_conflict ; then
-        if ! flannel_is_single_node; then
-            bail "Migrations from Weave to Flannel are not supported"
+        if [ -n "$DOCKER_VERSION" ]; then
+            bail "Migrations from Weave to Flannel are only supported with containerd"
```

Comment:
Could we add here, please migrate your host to use containerd and then, re-run this script? What do you think?

Reply:
maaaaaybe, but this is really a 'vendor needs to fix' problem and not a 'user can fix' problem if that makes any sense

the user can't migrate from docker->containerd on their own, they need the vendor to update the spec